superset/migrations/env.py

```diff
@@ -43,8 +42,9 @@
         "SQLite Database support for metadata databases will \
         be removed in a future version of Superset."
     )
-decoded_uri = urllib.parse.unquote(DATABASE_URI)
-config.set_main_option("sqlalchemy.url", decoded_uri)
+# Escape % chars in the database URI to avoid interpolation errors in ConfigParser
+escaped_uri = DATABASE_URI.replace("%", "%%")
```

Comment:
Do we not need to `urllib.parse.unquote` anymore?

Reply:
See my comment above; this was never needed, it was actually a bit of a dodgy hack.